vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -0,0 +1,63 @@
+# How to customize Vizro-AI
```

Comment:
It really is a guide on how to customize the model, so I would probably clearlt label it like that

Reply:
(see also now the chosen title in the TOC)

Reply:
I went with "How to customize model usage"